typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,16 @@
+const fullName: string = 'Jeffrey Mejia Cortez';
+const isCool: boolean = true;
+const totalPets: number = 0;
+
+console.log('value of fullName: ', fullName);
+console.log('value of isCool: ', isCool);
+console.log('value of totalPets:', totalPets);
+console.log('typeof fullName: ', typeof fullName);
+console.log('typeof isCool: ', typeof isCool);
+console.log('typeof totalPets: ', typeof totalPets);
+
+const unknown: null = null;
+const never: undefined = console.log('value of unknown: ', unknown);
```

Comment:
`never` should not be getting assigned any value. It should just be created and typed as `undefined`. No need to assign anything to it!